<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guestManagers</class>
 <widget class="QDialog" name="guestManagers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Наши менеджеры</string>
  </property>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="twOrg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>64</y>
     <width>641</width>
     <height>223</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="teResult">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>344</y>
     <width>321</width>
     <height>81</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSelectAll">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>304</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Получить данные</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Список наших менеджеров:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
